### ISSUE TITLE
Introspection should comment out full Enum definition if no valid values present

### ISSUE THREAD

**Issue by janpio** [prisma/prisma#3752]
I really like the introspection in general. We just have one small problem
It creates this enum which is invalid and `prisma validate` throws the error `Error validating: An enum must have at least one value.`.  If the enum is empty on introspection, the definition should be commented out too. 
```js
enum enum_vendors_menuPriceBand {
  // £ @map("£")
  // ££ @map("££")
  // £££ @map("£££")
}
```

I know this is probably a bad enum to have and will be removed eventually, but it would be nice if it didn't break our project every time we introspect.

_Originally posted by @Swapnull in https://github.com/prisma/prisma/issues/3659#issuecomment-698183613_

**Comment by janpio** [comment]
@Swapnull, turned your comment into its own issue so we can debug it:

> but it would be nice if it didn't break our project every time we introspect.

Have you tried giving the 3 values "proper" names (e.g. `pound`, `poundpound` and `poundpoundpound`) and uncommented these lines? As far as I know that should make them usable _and_ these names should be kept if you run `prisma introspect` again.

**Comment by tomhoule** [comment]
Pending on a principled approach to identifiers in PSL.

**Comment by janpio** [comment]
We prefer to have a validation error here so the user is aware and can fix it manually - instead of introducing complicated renaming logic.
